src/Sarif.UnitTests/ExtensionsTests.cs

```diff
@@ -38,31 +38,31 @@ public void Extensions_PropertValue_SetValue()
         [TestMethod]
         public void Extensions_IsNewline_CarriageReturn()
         {
-            Assert.IsTrue(Extensions.IsNewline('\r'));
+            Assert.IsTrue(ExtensionMethods.IsNewline('\r'));
```

Comment:
>ExtensionMethods.IsNewline('\r') [](start = 26, length = 32)

Urk! Make this an actual extension method, so you can write

if (ch.IsNewline())
{
    .
}
```

Reply:
what the heck, why wasn't this an extension method?? :)

---
In reply to: [136153212](https://github.com/Microsoft/sarif-sdk/pull/612#discussion_r136153212) [](ancestors = 136153212)